rq/worker.py

```diff
@@ -737,6 +737,10 @@ def fork_work_horse(self, job, queue):
             self._horse_pid = child_pid
             self.procline('Forked {0} at {1}'.format(child_pid, time.time()))
 
+    def get_heartbeat_ttl(self, job, elapsed_execution_time):
```

Comment:
Instead of passing `elapsed_execution_time` around, I think it makes sense for the worker to keep track of the start of job execution time. Can we make this change?